cuml/coordinate_descent_demo.ipynb

```diff
@@ -128,19 +129,30 @@
     "#### Fit"
    ]
   },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "\n",
+    "from sklearn.linear_model import Lasso"
+   ]
+  },
   {
    "cell_type": "code",
    "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
     "%%time\n",
-    "ols_sk = skLasso(alpha=np.array([learning_rate]), \n",
-    "                 fit_intercept = True, \n",
-    "                 normalize = False,\n",
-    "                 max_iter = 1000,\n",
-    "                 selection=algorithm,\n",
-    "                 tol=1e-10)\n",
+    "ols_sk = Lasso(alpha=np.array([learning_rate]),\n",
```

Comment:
keeping the name of the two functions might get confusing for the user if they are not careful or if they dont run the cells right, I think it would be better if we either just import sklearn and cuml in the first cell and then call the function using the whole path `sklearn.linear_model.Lasso` or use different names like `skLasso`

Reply:
I understand and partially agree with your comments, but please notice that SKL and cuML versions of the code are differentiated in the headers before the cells.

I thought about prioritising the following: making clear that the names in both libraries is the same, so porting the code is super easy (in some scenarios, it is only needed to change the import).

In addition to that, these notebooks are not only useful to compare SKL and cuML, but they will also be taken by the community as the base to multiple cuML notebooks. I think it is better to keep on naming Lasso class as Lasso, instead of making the community to think that the right naming is cuLasso, or something similar.

Could you please reconsider keeping my approach?

(Not fixed yet in local)

Reply:
I understand that, my concern arises from the senario where the user is not careful. The notebooks are also meant to be a tool given to the user where they can alter parameters, observe and explore the cuml models. If the user is not careful then they might run the wrong model. 
If we use the entire path while calling the model function (`sklearn.linear_model.Lasso`) it will help to showcase that both the library names are same and the only change required to switch is to replace sklearn with cuml.
If you would like to keep the names same we can also move the import statements inside the cell creating the model to ensure the right model is called.
@JohnZed what do you think?

Reply:
Hi @Salonijain27 , @JohnZed ,

I will try what Saloni suggested and submit a new commit:

> If you would like to keep the names same we can also move the import statements inside the cell creating the model to ensure the right model is called.

Reply:
Done and pushed for review!